### ISSUE TITLE
Select - Enhance Target Size WCAG 2.5.5

### ISSUE THREAD

**Issue by julauxen** [grommet/grommet#7624]
Select target size's hight is 36px plus 1 px padding and to achieve 2.5.5 Target Size (Enhanced) it should be 44px of height.
